tests/framework/utils/k8s_helper.go

```diff
@@ -644,3 +646,43 @@ func (k8sh *K8sHelper) IsRookInstalled() bool {
 
 	return false
 }
+
+//GetRookLogs captures logs from specified rook pod and writes it to specified file
+func (k8sh *K8sHelper) GetRookLogs(podAppName string, namespace string, testName string) {
+	logOpts := &v1.PodLogOptions{}
+	listOpts := metav1.ListOptions{LabelSelector: "app=" + podAppName}
+
+	podList, err := k8sh.Clientset.Pods(namespace).List(listOpts)
+	if err != nil {
+		logger.Errorf("Cannot get logs for app : %v in namespace %v, err: %v", podAppName, namespace, err)
+		return
+	}
+
+	for _, pod := range podList.Items {
+		podName := pod.Name
+		logger.Infof("getting logs for pod : %v", podName)
+		res := k8sh.Clientset.Pods(namespace).GetLogs(podName, logOpts).Do()
+		rawData, err := res.Raw()
+		if err != nil {
+			logger.Errorf("Cannot get logs for app : %v in namespace %v, err: %v", podAppName, namespace, err)
```

Comment:
i think you'd be able to do a `continue` instead of a `return` here.  if 1 pod fails, the next pod may still work?

if you did that, there would be an even stronger argument for logging `podName` instead of `podAppName` in the error log statement.